order-service/src/main/java/com/rushcrew/order/domain/order/OrderReservation.java

```diff
@@ -0,0 +1,113 @@
+package com.rushcrew.order.domain.order;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.UUID;
+
+import com.rushcrew.order.domain.order.enums.ReservationStatus;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "p_order_reservation", schema = "order_schema")
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
+@Builder
```

Comment:
```suggestion
@Builder(access = AccessLevel.PRIVATE)
```

create메서드가 생성해주니 
접근제어자를 수정해줘도 좋겠네요!